### ISSUE TITLE
Is there a LayerNormalization Plugin ?

### ISSUE THREAD

**Issue by taomiao** [NVIDIA/TensorRT#210]
## Environment

**TensorRT Version**: 6.0.1
**GPU Type**: 
**Nvidia Driver Version**: 
**CUDA Version**: 
**CUDNN Version**: 
**Operating System + Version**: 
**Python Version (if applicable)**: 
**TensorFlow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if container which image + tag)**:

**Comment by rmccorm4** [comment]
Hi @taomiao,

Can you try exporting a simple model (Pytorch/TF) with LayerNormalization to ONNX, try to convert ONNX->TRT with `trtexec --onnx=model.onnx`, and see what happens?

**Comment by sushilkhadkaanon** [comment]
Hi @rmccorm4 , @taomiao , I have LayerNormalization in my onnx model and I have to run it in TensorRT version 8.4.1 (can't use the latest one due to restrictions). How do I solve this issue? After exporting LayerNormalization to ONNX and to tensorrt, how do I integrate it with my actual model?

Please help me on this, Thanks

`
[09/18/2024-09:34:49] [I] [TRT] No importer registered for op: LayerNormalization. Attempting to import as plugin.
[09/18/2024-09:34:49] [I] [TRT] Searching for plugin: LayerNormalization, plugin_version: 1, plugin_namespace: 
[09/18/2024-09:34:49] [E] [TRT] parsers/onnx/ModelImporter.cpp:773: While parsing node number 2 [LayerNormalization -> "x.3"]:
[09/18/2024-09:34:49] [E] [TRT] parsers/onnx/ModelImporter.cpp:774: --- Begin node ---
[09/18/2024-09:34:49] [E] [TRT] parsers/onnx/ModelImporter.cpp:775: input: "input.1"
input: "onnx::LayerNormalization_7"
input: "onnx::LayerNormalization_8"
output: "x.3"
name: "LayerNormalization_802"
op_type: "LayerNormalization"
attribute {
  name: "axis"
  i: -1
  type: INT
}
attribute {
  name: "epsilon"
  f: 1e-05
  type: FLOAT
}

[09/18/2024-09:34:49] [E] [TRT] parsers/onnx/ModelImporter.cpp:776: --- End node ---
[09/18/2024-09:34:49] [E] [TRT] parsers/onnx/ModelImporter.cpp:778: ERROR: parsers/onnx/builtin_op_importers.cpp:4890 In function importFallbackPluginImporter:
[8] Assertion failed: creator && "Plugin not found, are the plugin name, version, and namespace correct?"
[09/18/2024-09:34:49] [E] Failed to parse onnx file
[09/18/2024-09:34:49] [I] Finish parsing network model
[09/18/2024-09:34:49] [E] Parsing model failed
[09/18/2024-09:34:49] [E] Failed to create engine from model or file.
[09/18/2024-09:34:49] [E] Engine set up failed`

**Comment by ntvuongg** [comment]
.
